app/models/checkin.rb

```diff
@@ -18,7 +18,14 @@ def self.students_at_hackathon
         return []
       end
   		Checkin.where(room_id: check_in.id).where(is_mentor: false)
-  	end
+    end
+    
+    def self.students
+      if Student.count == 0
```

Comment:
What happens when student.count is greater than zero or less than zero?

Reply:
Greater than zero - it returns all the students that have checked in and less than zero - there's no case for it. Should I add a case or should be we be ok since there is physically no way for the  student count to go below 0 unless we set it like that ourselves?

Reply:
We should be fine then I'll merge